pilot/pkg/config/kube/ingress/controller.go

```diff
@@ -21,6 +21,7 @@ import (
 	"fmt"
 	"sort"
 	"sync"
+	"sync/atomic"
```

Comment:
shouldn't it be "go.uber.org/atomic"?

Reply:
I think we can use `sync/atomic` in go1.18

Reply:
Isn't atomic.Bool introduced in 1.19, while istio still use 1.18?